rust/tests/command_optimize.rs

```diff
@@ -493,7 +493,7 @@ async fn test_commit_info() -> Result<(), Box<dyn Error>> {
     assert_eq!(last_commit["readVersion"], json!(version));
     assert_eq!(
         last_commit["operationParameters"]["targetSize"],
-        json!(2_000_000)
+        json!("2000000")
```

Comment:
I'd also like to see a test for `partitionValues`. Could you add that?

Reply:
Sorry, could you please clarify your request? I ask because "partitionValues" is not part of the commit info and operating params that have been changed in this PR. 
Perhaps you want to check that "partitionBy" as part of the operating params is built correctly?
 
If you want exactly "partitionValues", I can figure out how to check that. But it looks like it should be in a separate PR.

Reply:
Sorry, could you please clarify your request? I ask because "partitionValues" is not part of the commit info and operating params that have been changed in this PR.
Perhaps you want to check that "partitionBy" as part of the operating params is built correctly?

If you want exactly "partitionValues", I can figure out how to check that. But it looks like it should be in a separate PR.

Reply:
Yes, sorry. I meant "partitionBy".

Reply:
Added separated unit test, because the most of commit info sections doesn't pass operation info. So they doesn't contains operational parameters which I can use to check.